### PR TITLE
fix: Update colors in `DescriptionHeaderView`

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR I update the colors in `DescriptionHeaderView`. In addition to that I made a couple of changes in `ConfirmEmailViewControllerTests` so from now on we use `SnapshotTesting` instead of Facebook's library. 

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift [hunk 1]
```diff
@@ -52,13 +56,20 @@ extension UITableView {
...
+// MARK: - ConfirmEmailViewController
+
 final class ConfirmEmailViewController: SettingsBaseTableViewController {
+
+    // MARK: - Properties
+
     fileprivate weak var userProfile = ZMUserSession.shared()?.userProfile
     weak var delegate: ConfirmEmailDelegate?
-
+    typealias SettingsAccountSectionEmail = L10n.Localizable.Self.Settings.AccountSection.Email.Change
```
Comment (SzymonMrozek) [inline wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift]: shouldn't that type contain some keyword pointing it's about localization/strings?

**Review by SzymonMrozek — APPROVED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift]: Good point! What would be your suggestion?

Comment (SzymonMrozek) [reply wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift]: I don't know if there is any convention in the repo, but maybe something like,: 
`SettingsAccountSectionEmailLocalizable` or if that's too long `SettingsEmailLocalizable` ?

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ConfirmEmailViewController.swift]: I don't think we have any convention at the moment. I think I will go with the first option!

**Pull request merged**